LiveSched/src/main/java/dev/coms4156/project/livesched/LiveSchedApplication.java

```diff
@@ -31,7 +37,100 @@ public static void main(String[] args) {
    */
   @Override
   public void run(String[] args) {
-    //to be added
+    for (String arg : args) {
+      if (arg.equals("setup")) {
+        myFileDatabase = new MyFileDatabase(1, TASK_FILE_PATH, RESOURCE_TYPE_FILE_PATH);
+        setupDataFile();
+        System.out.println("System setup completed.");
```

Comment:
we should update these to loggers after iteration 1

Reply:
Yes!